missions/W4/M2/extract.py

```
@@ -0,0 +1,12 @@
+from app.core.constants import VehicleType
+from app.core.extractor import Extractor
+from app.utils.logging import Logger
```

Ruff (0.8.2)

`app.utils.logging.Logger` imported but unused

Remove unused import: `app.utils.logging.Logger`

(F401)